src/Template/Pages/home.ctp

```diff
@@ -21,10 +21,10 @@ use Cake\Network\Exception\NotFoundException;
 $this->layout = false;
 
 if (!Configure::read('debug')):
-    throw new NotFoundException('Please replace Pages/home.ctp with your own version.');
+    throw new NotFoundException(__d('cake', 'Please replace Pages/home.ctp with your own version.'));
 endif;
 
-$cakeDescription = 'CakePHP: the rapid development php framework';
+$cakeDescription = __d('cake', 'CakePHP: the rapid development php framework');
```

Comment:
You would need to make all names {n} parts as they are not subject of translation, ever.